### PR TITLE
add sourceMaps

### PR DESCRIPTION
**Pull request by peter-mouland**
also : 
 * removed css test in webpack as there are no css files
 * updated post css and autoprefixer to recommended method

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -14,14 +16,18 @@ module.exports = [
     },
     module: {
       loaders: [
-        { test: /\.scss$/, loader: ExtractTextPlugin.extract("css!autoprefixer?browsers=last 2 versions!sass") },
-        { test: /\.css$/, loader: ExtractTextPlugin.extract("css!autoprefixer?browsers=last 2 versions") },
```
Comment (nowarkdev) [inline webpack.config.js]: Where are we configuring autoprefixer to specify the last 2 versions now?
@bdo @rogerfranklin2

Comment (peter-mouland) [inline webpack.config.js]: I thought latest 2 versions is default...

after some research, this has changed slightly to being `> 1%, last 2 versions, Firefox ESR` is default.

want me to put it back in?

Comment (bdo) [inline webpack.config.js]: Thanks for checking, I think the defaults are fine actually.

**Comment by bdo** [comment]
:+1:

**Comment by alexflorisca** [comment]
:+1:

**Pull request merged**